src/parsec_service/stability.md

```diff
@@ -0,0 +1,68 @@
+|--------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------|
+| Communication with clients     | Requests and responses must be understood correctly by different versions of the service/clients                                                       | The Wire Protocol is [stable by design](../parsec_client/wire_protocol.md#wire-protocol-versions).                                                                                                                                                                  | TODO           |
+| Authenticators                 | Authentication requests sent by future stable versions of Parsec should be understood by the same version of the authenticators present on the system. | Unix Peer Crendentials: based on top of stable C standard library APIs. JWT SVID: authentication is based on the [SPIFFE Workload API](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE_Workload_API.md) which is a standard and should remain stable. | TODO           |
+| CLI invocation                 | Old CLI invocations of Parsec should still work with for future stable Parsec versions.                                                                | CLI flags should not be removed. New flags should be optional.                                                                                                                                                                                                      | TODO           |
+| Configuration file             | Old configuration files should still work with future stable Parsec versions, with the same default for optional options.                              | Configuration options should not disappear in future stable Parsec versions. Configuration defaults should remain the same. New options should be optional.                                                                                                         | TODO           |
+| Key mappings                   | Old key mappings should still be read correctly by future stable Parsec versions.                                                                      | The way the KIM persistently stores the mappings need to be backward-compatible in regards with reading/writing.                                                                                                                                                    | TODO           |
```

Comment:
I guess the trickier question is how we actually handle breaking-change transitions here, but that's probably for another document.

Reply:
Yes I will open a specific issue for each of those points where we can discuss that.